app/src/androidTest/java/com/github/h3lp3rs/h3lp/SideBarTest.kt

```diff
@@ -74,7 +68,7 @@ class SideBarTest {
         openDrawerLayout()
         onView(withId(R.id.nav_view))
             .perform(NavigationViewActions.navigateTo(R.id.nav_home))
-        Thread.sleep(500)
+        Thread.sleep(WAIT_UI)
```

Comment:
What is this!!
I know you did not do this, but how did such atrocious code end in our codebase? :O